### PR TITLE
Added burn endpoint

### PR DESCRIPTION
**Pull request by Cedev**
Example usage
    https://burn-boise-pr-4.herokuapp.com/burn/83704/open
    https://burn-boise-pr-4.herokuapp.com/burn/83704/fireplace
    https://burn-boise-pr-4.herokuapp.com/burn/83704/woodstove
    https://burn-boise-pr-4.herokuapp.com/burn/83704/certified

 * Added mapping from zip codes to regulatory areas
 * Added regulatory thresholds
 * Index page is now aware of regulatory thresholds

### CODE REVIEW

File: burn_boise.rb [hunk 1]
```diff
@@ -11,31 +12,125 @@ def get_aqi(zipcode)
   aqi
 end
 
+class Regulations
```
Comment (0xtobit) [inline burn_boise.rb]: Might be better to move this to `lib/regulations.rb` and `require` it at the top

Comment (Cedev) [reply burn_boise.rb]: It sneakily uses `get_aqi`

**Comment by 0xtobit** [comment]
~I'm getting `Forbidden` on the example links~

**Comment by Cedev** [comment]
Some nice tests:

Orange right now (61 aqi): https://burn-boise-pr-4.herokuapp.com

White for CDL (no rules): https://burn-boise-pr-4.herokuapp.com/?zipcode=83814

False for open burning right now (61 aqi): https://burn-boise-pr-4.herokuapp.com/burn/83704/open

True for fireplace burning right now (61 aqi): https://burn-boise-pr-4.herokuapp.com/burn/83704/fireplace

Null for CDL right now (no rules): https://burn-boise-pr-4.herokuapp.com/burn/83814/open

If you get `Forbidden` click in url bar and press enter to reload without CORS.

File: burn_boise.rb [hunk 2]
```diff
@@ -11,31 +12,125 @@ def get_aqi(zipcode)
...
   zipcode = params[:zipcode] || "83704"
   aqi = get_aqi(zipcode)
 
-  # http://www2.deq.idaho.gov/air/AQIPublic/Ordinance/Index/3
-  case aqi
-  when 0..59
+  prohibited = Regulations.prohibited_reactors(zipcode, aqi)
+
+  if prohibited.nil?
+    background_color = 'white'
```
Comment (0xtobit) [inline burn_boise.rb]:
```suggestion
    background_color = 'dark-gray'
```

**Pull request merged**